pull_request_template.md

```diff
@@ -1 +1,2 @@
-- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests
+- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests.
+- [ ] I opened a pull request on the Node playground if the changes are breaking.
```

Comment:
Can you add the link to your PR on kili playground node in your gitlab MR ? As a comment ? Thanks

Reply:
https://github.com/kili-technology/kili-playground-node/pull/9